### ISSUE TITLE
Invalid input JSON when calling a sub orchestration

### ISSUE THREAD

**Issue by Kimmen** [Azure/azure-functions-durable-extension#86]
When calling a SubOrchestration we are getting invalid json as input. 

The input type has two string properties:
```
class Input
{
    public string Id { get; set; }
    public string Name { get; set; }
}
```
and we are calling the sub orchestration as 
`context.CallSubOrchestratorAsync<Output>("SubOrchestration", new Input {Id = "Id1", Name = "Name1"});`

When calling GetInput<Input>() we get an error trying to convert the json string to our input type.  When we look at the JSON string (GetRawInput) there's invalid/unexpected json:
`"\"[{\\\"Id\\\":\\\"Id1\\\",\\\"Name\\\":\\\"Name1\\\"}]\"" ` 
(we printed it using the immediate window in VS2017). 

Our workaround right now is to use fix this json ourselves then deserializing it:
```
var json = this.context.GetRawInput();
return Newtonsoft.Json.JsonConvert.DeserializeObject<T>(json.Substring(2, json.Length - 4).Replace("\\", ""));
```

**Comment by cgillum** [comment]
This is a duplicate of https://github.com/Azure/azure-functions-durable-extension/issues/76, which is fixed in the dev branch. I'll close this issue once the new release is uploaded to nuget.org.

**Comment by cgillum** [comment]
This is fixed in the 1.1.0-beta2 release.
